src/app/loginComponents/accounts/accounts.component.ts

```diff
@@ -57,6 +57,6 @@ export class AccountsComponent extends Base implements OnInit {
   setAccountsTab(tabName: string) {
     // Adding the & symbol causes the URL to be parsed in an unexpected way.
     // TODO: Change replace to replaceAll once we use EC2021
-    this.location.replaceState('accounts?tab=' + tabName.replace('&', 'and'));
+    this.location.replaceState('accounts?tab=' + tabName.replace(/&/g, 'and'));
```

Comment:
This works.  Technically, if we wanted to be 100% robust and allow full freedom of naming, instead of the bandaid `&` -> `and`, we should fully urlencode `tabname` via `encodeURIComponent()` and change the `and`s back to `&`s in `validTabs`.  But I'm not sure it's worth the trouble in this case.

Reply:
I was trying to think of a bug but since we constrain the input it shouldn't happen? If you think there's a bug we should open it though.

Reply:
Yeah, I think it's ok for now, no bugs in the current configuration (that I can see).